www-apps/bugzilla/bugzilla-5.0.6.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit webapp eutils # should not depend on a web server
+
+DESCRIPTION="Bugzilla is the Bug-Tracking System from the Mozilla project"
+SRC_URI="https://ftp.mozilla.org/pub/mozilla.org/webtools/${P}.tar.gz"
+HOMEPAGE="https://www.bugzilla.org"
+
+LICENSE="MPL-2.0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="mysql postgres sqlite test" #modperl graphviz are not suitable for USE as per https://devmanual.gentoo.org/general-concepts/use-flags/
+RESTRICT="!test? ( test )" #why restrict tests ?
+REQUIRED_USE=" || ( mysql postgres sqlite )"
+
+DEPEND="
+	dev-lang/perl
+	>=dev-perl/CGI-3.510.0
+	>=dev-perl/DBI-1.614.0
+	>=dev-perl/TimeDate-2.230.0
+	>=dev-perl/DateTime-0.750.0
+	>=dev-perl/DateTime-TimeZone-1.640.0
+	virtual/perl-Digest-SHA
+	>=dev-perl/Email-MIME-1.904.0
+	>=dev-perl/Email-Sender-1.300.11
+	>=dev-perl/JSON-XS-2.10.0
+	>=dev-perl/List-MoreUtils-0.320.0
+	>=dev-perl/Math-Random-ISAAC-1.0.1
+	>=dev-perl/Template-Toolkit-2.240.0
+	>=dev-perl/URI-1.550.0
+	postgres? ( >=dev-perl/DBD-Pg-1.49 )
+	mysql? ( >=dev-perl/DBD-mysql-4.0.5 )
+	sqlite? ( >=dev-perl/DBD-SQLite-1.290.0 )
+	test? ( dev-perl/Pod-Coverage )
+"
+need_httpd_cgi
+
+src_prepare() {
+	# Get a rid of the bzr files
+	rm -rf .bzr* || die
+
+	# Remove bundled perl modules -> not needed: this directory only contains a README file and is needed for checksetup.pl with sqlite
+#	rm -rf lib/* || die
+
+	# Fix a template error to pass 009bugwords.t
+	sed -i -e 's:a bug:\[\% terms.abug \%\]:'\
+		template/en/default/pages/release-notes.html.tmpl || die
+
+	# Add dummy coverage for two subroutines that are not covered
+	echo -e "\n=over\n\n=item write_text\n\n=back" >> Bugzilla/Util.pm || die
+	echo -e "\n=over\n\n=item read_text\n\n=back" >> Bugzilla/Util.pm || die
+
+	# Fix Unescaped left brace in regex is passed through in regex; marked by <-- HERE in m/\${ <-- HERE [^}]+}/ at t/010dependencies.t line 74. Replaces patch bugzilla-5.0.3-leftbrace.patch
+	sed -i -e 's:\$used =~ s\#\\\${\[^}\]+}\#\[^\:\]+\#:\$used =~ s\#\\\$\\{\[^}\]+}\#\[^\:\]+\#:' \
+		t/010dependencies.t || die
+
+	# See https://bugzilla.mozilla.org/show_bug.cgi?id=1588175 and https://github.com/Perl/perl5/issues/17271
+	sed -i -e 's:\$var =~ tr\/\\x{202a}-\\x{202e}\/\/d:\#&:' \
+		Bugzilla/Util.pm || die
+
+	default
+}
+
+src_test() {
+	perl -I. runtests.pl || die
+}
+
+src_install() {
+	webapp_src_preinst
+
+	insinto "${MY_HTDOCSDIR}"
+	doins -r . || die
+#	doins "${FILESDIR}"/bugzilla.cron.{daily,tab} 
+#	webapp_hook_script "${FILESDIR}"/reconfig # I do not understand these two lines at the moment
```

Comment:
Needs to be resolved before merging so it won't break existing installations.

Reply:
Those are optionnal post-install config. Existing installation with these files already present should not be affected. Shouldn't it be up to the user to update their post-install configurations ?